x/stake/keeper/validator.go

```diff
@@ -246,13 +254,69 @@ func (k Keeper) UpdateValidator(ctx sdk.Context, validator types.Validator) type
 			bz := k.cdc.MustMarshalBinary(validator.ABCIValidator())
 			store.Set(GetTendermintUpdatesKey(validator.Owner), bz)
 		}
-
 	}
 
 	k.SetValidator(ctx, validator)
 	return validator
 }
 
+// updateCliffValidator determines if the current cliff validator needs to be
```

Comment:
I like to call this, "maybeUpdateCliffValidator", which implies that it only happens conditionally.
The comment could be better, as it does more than determine the conditional.

Reply:
Agreed more appropriate nomenclature could be defined and standardized, but I'm not convinced `Ensure*` is it?

Reply:
`ensure` implies a post-condition for the function, which is accurate

Reply:
(then we should comment explaining what the postcondition is)

Reply:
At first it seemed to me, `ensure` implies something must be done (in this, now old, case would imply that cliff must be updated, even though it's possible for it not to).

Now that I think about it more, it kinda makes sense.

Reply:
@alexanderbez Did we reach consensus on updating the name to `ensureCliffValidator`?